merchants/verkkokauppa/payment/requests.py

```diff
@@ -64,6 +71,49 @@ def get_payment(order_id: UUID, namespace: str, get=_get) -> Optional[Payment]:
         raise GetPaymentError("Payment retrieval failed") from e
 
 
+def get_refund_status(
+    order_id: UUID, namespace: str, get=_get
+) -> Optional[RefundStatusResult]:
+    try:
+        with ExternalServiceMetric(
+            METRIC_SERVICE_NAME, "GET", "/payment/admin/refund-payment/{order_id}"
+        ) as metric:
+            response = get(
+                url=urljoin(_get_base_url(), f"admin/refund-payment/{order_id}"),
+                headers={
+                    "api-key": settings.VERKKOKAUPPA_API_KEY,
+                    "namespace": namespace,
+                },
+                timeout=REQUEST_TIMEOUT_SECONDS,
+            )
+            metric.response_status = response.status_code
+
+        json = response.json()
+        errors = json.get("errors", [])
+
+        # API documentation says it returns 404 but in reality it
+        # returns 500 with error code. They have plans to fix this
+        # but just to avoid breaking things with unnotified changes we
+        # are handling both cases. 500 handling can be removed after
+        # webshop API is updated.
+        if (
+            response.status_code == 500
+            and errors
+            and errors[0].get("code") == "failed-to-get-refund-payment-for-order"
```

Comment:
'len(errors) > 0'

Reply:
As discussed in Slack, Python's truthful tests return False for empty lists so this should not cause index out of bounds error. However, because this check is not super obvious, I'll change it to `len(errors) > 0` as you suggested. It is more clear for everyone 👍